package.json

```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/ArrowSphere/nodejs-api-client.git"
   },
-  "version": "3.146.0",
+  "version": "3.147.0-rc.mla.2",
```

Comment:
Put the prod version here: 3.147.0

Reply:
Put the package.json & CHANGELOG.md in a separate commit.
Name the commit with that syntax: **chore(package): version 3.147.0**